.github/workflows/build-test.yml

```diff
@@ -288,6 +288,8 @@ jobs:
       with:
         key: download-cache
         path: /tmp/icu4x-source-cache
+    - name: Install rustfmt
```

Comment:
Can you test how much time formatting costs?

Reply:
With fmt:

```
real	0m18.767s
user	1m2.729s
sys	0m4.626s

real	0m19.310s
user	1m21.325s
sys	0m5.306s

real	0m19.624s
user	1m22.069s
sys	0m5.376s
```

Without fmt:

```
real	0m16.967s
user	1m12.534s
sys	0m0.914s

real	0m12.848s
user	0m48.084s
sys	0m0.730s

real	0m16.216s
user	0m57.994s
sys	0m0.824s
```

Reply:
cool thanks.